src/DirectWebServices/Shared/DWSConverters/RegionExposure/RegionExposureOptions.ts

```diff
@@ -0,0 +1,41 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Jedrzej Ruta
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type {
+    MorningstarConverterOptions,
+    MorningstarMetadata
+} from '../../../../Shared';
+import type { Message } from '../../../InvestmentsConnector/InvestmentsJSON';
+import type RegionExposureJSON from './RegionExposureJSON';
+
+export interface RegionExposureOptions extends MorningstarConverterOptions {
+    json: RegionExposureJSON.RegionExposureResponse;
+}
+
+export interface RegionExposureMetadata extends MorningstarMetadata {
```

Comment:
The same as above with the `RegionExposureConverterMetadata`. Additionaly, nice work leaving the `MorningstarMetadata`.

This made me thinking that my previous implementation of the whole metadata logic is not perfect when it comes to types. So I prepared some corrections. My question is, could we maybe implement them here, instead of me creating a new branch for that purpose alone?

... I will take this as a yes :D.